app/ui/src/app/app.module.ts

```diff
@@ -23,12 +21,14 @@ import { TourNgxBootstrapModule } from 'ngx-tour-ngx-bootstrap';
 import { NotificationModule } from 'patternfly-ng';
 import { DataMapperModule } from '@atlasmap/atlasmap.data.mapper';
 
+import { environment } from '../environments/environment';
 import { AppComponent } from './app.component';
 import { AppRoutingModule } from './app.routing';
 import { CanDeactivateGuard, SyndesisCommonModule } from './common';
 import { UserService } from './common/user.service';
 import { ConfigService } from './config.service';
-import { StoreModule } from './store/store.module';
+import { StoreModule as LegacyStoreModule } from './store/store.module';
```

Comment:
This is good so that we can prevent breaking existing code that depends on this store (most of the app, atm). 👍